src/foam/comics/v2/BasicEnabledActionsAuth.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v2',
+  name: 'BasicEnabledActionsAuth',
+  implements: [
+    'foam.comics.v2.EnabledActionsAuth'
+  ],
+  javaImports: [
+    'foam.util.SafetyUtil'
+  ],
+
+  documentation: `
+    String arrays of permissions that will be checked on each CRUD action through availablePermissions
```

Comment:
Is it really an array?

Reply:
Old documentation code, updated after joels comments.